hack/update-pkg-manifest.sh

```diff
@@ -13,82 +13,91 @@
+  generate_csv "${deploy_path}/${rh_bundle_version}" "${rh_operator_image}"
+  combine_files "${deploy_path}/${rh_bundle_version}" "${rh_bundle_version}"
+}
+
+ensure_unique_deployment() {
+  if [ -d "${1}" ]; then
+    echo "Folder ${1} already exists. Please increase the version or remove the folder manually." >&2
     exit 1
-fi
-rm -rf ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}
-"$opsdk" generate kustomize manifests -q --verbose
-"$kstomize" build config/manifests | "$opsdk" generate packagemanifests -q --version ${RH_BUNDLE_VERSION} ${RH_PKG_MAN_OPTS} --output-dir ${DEPLOY_PATH} --input-dir ${DEPLOY_PATH} --verbose
-# cat ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/cockroach-operator.clusterserviceversion.yaml | sed -e "s+RH_COCKROACH_OP_IMAGE_PLACEHOLDER+${RH_COCKROACH_OP_IMG}+g" -e "s+RH_COCKROACH_DB_IMAGE_PLACEHOLDER+${RH_COCKROACH_DATABASE_IMAGE}+g" -e "s+CREATED_AT_PLACEHOLDER+"$(date +"%FT%H:%M:%SZ")"+g"> ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/csv.yaml
-cat ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/cockroach-operator.clusterserviceversion.yaml | sed "s+RH_COCKROACH_OP_IMAGE_PLACEHOLDER+${RH_COCKROACH_OP_IMG}+g; s+CREATED_AT_PLACEHOLDER+"$(date +"%FT%H:%M:%SZ")"+g"> ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/csv.yaml
-for v in $("$faq" -r '.CrdbVersions' "${REPO_ROOT}/crdb-versions.yaml" | cut -d ' ' -f2)
-do
-  vrs=${v//./_}
-  ENV_VAR_PLACEHOLDER="RH_COCKROACH_DB_IMAGE_PLACEHOLDER_${vrs}"
-  RH_COCKROACH_DATABASE_IMAGE_VERSION="registry.connect.redhat.com/cockroachdb/cockroach:${v}"
-  sed -i '' -e "s+${ENV_VAR_PLACEHOLDER}+${RH_COCKROACH_DATABASE_IMAGE_VERSION}+g" ${DEPLOY_PATH}/${RH_BUNDLE_VERSION}/csv.yaml
-done
-cd  ${DEPLOY_PATH}/${RH_BUNDLE_VERSION} && "$faq" -f yaml -o yaml --slurp '.[0].spec.install.spec.clusterPermissions+= [{serviceAccountName: .[2].metadata.name, rules: .[1].rules }] | .[0]' csv.yaml cockroach-database-role_rbac.authorization.k8s.io_v1_clusterrole.yaml cockroach-database-sa_v1_serviceaccount.yaml > cockroach-operator.v${RH_BUNDLE_VERSION}.clusterserviceversion.yaml
-shopt -s extglob
-rm -v !("cockroach-operator.v${RH_BUNDLE_VERSION}.clusterserviceversion.yaml"|"crdb.cockroachlabs.com_crdbclusters.yaml")
-shopt -u extglob
-# This is needed after csv generation
-cd ${REPO_ROOT}
-FILE_NAMES=(
-  config/manifests/bases/cockroach-operator.clusterserviceversion.yaml \
-)
-for YAML in "${FILE_NAMES[@]}"
-do
-   :
-   cat "${REPO_ROOT}/hack/boilerplate/boilerplate.yaml.txt" "${REPO_ROOT}/${YAML}" > "${REPO_ROOT}/${YAML}.mod"
-   mv "${REPO_ROOT}/${YAML}.mod" "${REPO_ROOT}/${YAML}"
-done
+  fi
+}
+
+generate_package_manifest() {
+  # Generate CSV in config/manifests and add boilerplate back (lost when regenerating)
+  operator-sdk generate kustomize manifests -q --apis-dir apis
+  hack/boilerplaterize hack/boilerplate/boilerplate.yaml.txt config/manifests/**/*.yaml
+
+  # TODO: packagemanifests is deprecated and scheduled to be remove in 2.0.0
+  kustomize build config/manifests | operator-sdk generate packagemanifests -q \
```

Comment:
How is the script using kustomize that is installed by bazel?  I am confused that you removed the arguments to the script that passed in the locations of kustomize.

Reply:
The `data` attribute for the rule ensures that the binaries are available (built if needed) in hack/bin, which is added to the path at the top of the script.